assets/dev/js/editor/editor-document.js

```diff
@@ -16,7 +17,7 @@ $e.modules.editor = {
 // TODO: Remove, BC.
 $e.modules.document = {
 	get CommandHistory() {
-		elementorCommon.helpers.softDeprecated(
+		deprecated(
```

Comment:
Please try to use the external module instead of importing it

Reply:
Handled.